compiler-core/src/language_server/compiler.rs

```diff
@@ -31,6 +31,9 @@ pub struct LspProjectCompiler<IO> {
     pub modules: HashMap<EcoString, Module>,
     pub sources: HashMap<EcoString, ModuleSourceInformation>,
 
+    // Information on compiled dependencies.
+    pub dependency_sources: HashMap<EcoString, ModuleSourceInformation>,
```

Comment:
Why is it that there's a second hashmap for dependencies rather than all being in one?

Reply:
Good point. I kinda just assumed we were using it in more places/would need to differentiate elsewhere but I guess not. Lemme just combine into 1

Reply:
That might be the case! I'm not sure.